content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md

```diff
@@ -103,6 +103,7 @@ The Nicla form factor has been specifically developed at Arduino® as a standard
 ### Assembly Overview
 ![Example of a typical solution for remote environmental sensing including an Arduino® Nicla Sense ME, Portenta H7 and battery. Notice the orientation of the battery's cable in the board's connector. ](assets/niclaSenseMEBattery.png)
 
+**Note** : The NTC pin on the battery connector is optional.
```

Comment:
```suggestion
**Note** : The NTC pin on the battery connector is optional. This is a feature allowing safer use and thermal shutoff of the PMIC. 
```
I checked with the task TC-414 details and thought maybe would be better to provide a little more of detail, and borrowed the word the from the task description. I think it should be enough with the suggested additional detail, but how do you see it?

Reply:
Thanks for bringing this up @TaddyHC . @martab1994 is the suggested change sufficent, or would you like additional information included about the NTC?

Reply:
That's perfect for me thanks @aliphys and @TaddyHC!